### DISCUSSION TITLE
How can I implement a text suggestion dropdown (similar to google)?

### DISCUSSION THREAD

**Discussion by pghanem** [ueberdosis/tiptap#3503] [General Questions]
Assuming I can generate the suggestions myself, what would be the best way to implement the dropdown? Notably, whats the best way to wire it to the editor so that, when selecting an option, it replaces the editor content? 

<img width="349" alt="examples" src="https://user-images.githubusercontent.com/34144710/206023021-a444a59d-d4bd-470b-a109-34ee3260659a.png">

**Comment by Lou99-GM** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
